pkg/neorpc/filters.go

```diff
@@ -20,6 +20,7 @@ type (
 	TxFilter struct {
 		Sender *util.Uint160 `json:"sender,omitempty"`
 		Signer *util.Uint160 `json:"signer,omitempty"`
+		Type   *string       `json:"type,omitempty"`
```

Comment:
OK, we actually don't have the ability to filter out transactions because these are in-block transactions (those that were accepted to chain). Thus, let's create a separate filter that is only for NotaryRequest events, e.g. some `NotaryRequestFilter`.